src/test/ui/consts/miri_unleashed/const_refers_to_static.rs

```diff
@@ -6,31 +6,31 @@
 use std::sync::atomic::AtomicUsize;
 use std::sync::atomic::Ordering;
 
-const BOO: &usize = { //~ ERROR undefined behavior to use this value
+const REF_INTERIOR_MUT: &usize = { //~ ERROR undefined behavior to use this value
```

Comment:
You don't like my useless variable names? :upside_down_face: 

If we'd had the `_` name feature back when I wrote the test I'dve used that

Reply:
I'd argue readable names are better even than `_` ;)